lntest/itest/lnd_test.go

```diff
@@ -2339,28 +2339,24 @@ func testOpenChannelAfterReorg(net *lntest.NetworkHarness, t *harnessTest) {
 	)
 
 	// Set up a new miner that we can use to cause a reorg.
-	args := []string{
-		"--rejectnonstd",
-		"--txindex",
-		"--nowinservice",
-		"--nobanning",
-	}
-	tempMiner, err := rpctest.New(
-		harnessNetParams, &rpcclient.NotificationHandlers{}, args,
+	tempLogDir := "./.tempminerlogs"
+	tempMiner, tempMinerCleanUp, err := lntest.NewMiner(
+		tempLogDir, "output_temp_btcd_miner.log",
```

Comment:
maybe the name of the test should be part of the file name here?

Reply:
Yes I think that's more indicative. Changed.